File: src/FSharpAux/Guid.fs
namespace FSharpAux


/// Wrapper to use System Guid
module Guid =
    
    open System
    
    /// Empty System.Guid
    let empty = Guid.Empty
    
    /// Parse a string to a System.Guid
    let ofString (s:string) =  new Guid(s)

    /// Creates a new unique Syste.Guid
    let create = Guid.NewGuid()

    

File: tests/FSharpAux.Mocha.Tests/fable/src/FSharpAux.Core/Guid.js
import { newGuid, parse } from "../../fable_modules/fable-library.4.0.6/Guid.js";

export const empty = "00000000-0000-0000-0000-000000000000";

export function ofString(s) {
    return parse(s);
}

export const create = newGuid();

